app/views/hosts/_operating_system.html.erb

```diff
@@ -9,26 +9,18 @@
 
 <div id='provisioning_method' <%= display? (@host.capabilities.size < 2)  %> >
   <%= field(f, :provision_method, :label => _('Provisioning Method'), :required => true) do
-      radio_button_f(f, :provision_method, :value=>'build', :checked=> @host.pxe_build?, :text=> _("Network Based"), :disabled => !@host.new_record?)+
-      radio_button_f(f, :provision_method, :value=>'image', :checked=> @host.image_build?, :text=> _("Image Based"), :disabled => !@host.new_record?)
+    Host::Managed::list_provision_methods.collect do |provision_method|
```

Comment:
To save the extra `provision_method_friendly_name` method call, this could return the hash and iterate with `.each do |method,friendly_name|` ?